### DISCUSSION TITLE
Vue js component in Block preview

### DISCUSSION THREAD

**Discussion by michaeljcoyne** [area17/twill#1281] [General]
Hi I am trying to get a vue component in the block preview.

I dont want a full compiled twill block handling the forms I just want to render it in the preveiw with props.

For some reason in the blocks blade file:
`
@php
    $speakers = app()->make(\App\Repositories\SpeakerRepository::class)->speakerBlock($block->browserIds('speakers'));
@endphp

\<example-component :speakers="{{json_encode($speakers)}}"></example-component>
`

Will not work. Is this.

Is it not possible to do it this way?

**Comment by pboivin** [comment]
Hi @michaeljcoyne, this should totally be possible in block previews.  In this context, `example-component` would be coming from your app's own javascript bundle. Make sure that you are loading Vue.js and related scripts in your `block.blade.php` layout. 

https://twill.io/docs/#previewing-blocks

**Reply by michaeljcoyne** [reply]
I @pboivin 

Ok I have done all this so was wondering why it wasn't working as expected. I just have a blade component extending the front end one. It displays front end but not on the preview.

**Comment by michaeljcoyne** [comment]
Oh it was working all along except it was browser cache.
